Assets/Scripts/Layers/Properties/PropertyData/LayerURLPropertyData.cs

```diff
@@ -2,19 +2,33 @@
 using System.Collections.Generic;
 using System.Runtime.Serialization;
 using Newtonsoft.Json;
+using UnityEngine.Events;
 
 namespace Netherlands3D.Twin.Layers.Properties
 {
     [DataContract(Namespace = "https://netherlands3d.eu/schemas/projects/layers/properties", Name = "Url")]
     public class LayerURLPropertyData : LayerPropertyData, ILayerPropertyDataWithAssets
     {
-        [DataMember] public string url = "";
-        
+        [DataMember] private Uri data;
```

Comment:
Door de field name te wijzigen breek je backwards-compatibility met eerdere versies van de project bestanden, in die project bestanden heet het veld nu 'url', maar na jouw change 'data'. Voor backwards compatibility zou ik de field url blijven noemen